src/Domain.LinnApps/SernosSequence.cs

```diff
@@ -5,7 +5,5 @@ public class SernosSequence
         public string Sequence { get; set; }
 
         public string Description { get; set; }
-
-        public string DecrementRule { get; set; }
```

Comment:
I don't remember adding this field and it didn't seem used so I took it out. Any objections let me know